style.css

```diff
@@ -123,3 +126,221 @@ p {
   line-height: 32px;
   text-align: center;
 }
+
+.card {
+  height: 350px;
+  background-color: #ebf0ee;
+  flex-direction: column;
+  justify-items: center;
+  justify-content: space-around;
+  align-content: center;
+  border-radius: 6px;
+  display: grid;
+  margin-top: 20px;
+  margin-right: 25px;
+  margin-left: 25px;
+}
+
+h2 {
+  text-align: center;
+  color: #344534;
+  margin-top: 5%;
+}
+
+.logo-img {
+  margin-left: 5%;
+  margin-bottom: 5%;
+}
+
+.logo-img1 {
+  margin-left: 5%;
+  margin-bottom: 5%;
+}
+
+.btn2 {
+  height: 30px;
+  border: none;
+  border-radius: 4px;
+  font-weight: 600;
+  font-size: 12px;
+  margin: 5px 5px;
+  padding: 10px 17px;
+  color: #36b37e;
+  background-color: #fff;
+}
+
+.last-p {
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+  margin-right: 10px;
+  margin-left: 10px;
+  margin-top: 60px;
+  margin-bottom: 5%;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  margin-right: 10px;
+  margin-left: 10px;
+  align-items: center;
+}
+
+form input {
+  margin-bottom: 5%;
+  border: 1px solid gray;
+  width: 100%;
+  padding: 5px 5px;
+}
+
+textarea {
+  width: 100%;
+  height: auto;
+}
+
+.hover:hover {
+  border: 3px solid #36b37e;
+}
+
+.link {
+  text-decoration: none;
+  text-align: center;
+  font-size: 24px;
+  color: #42526e;
+  margin-left: 30%;
+}
+
+.social_links1 {
+  display: flex;
+  list-style: none;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  margin-left: 10%;
+}
+
+.img2 {
+  background-image: url(images/illustration-contact-form.svg);
+  background-repeat: no-repeat;
+  background-color: transparent;
+  background-size: cover;
+  width: 92.14px;
+  height: 131.71px;
+  position: absolute;
+  right: 0;
+  bottom: -46px;
+}
+
+.enginn {
+  position: relative;
+}
+
+#about-me {
+  position: relative;
+}
+
+.img3 {
+  background-image: url(images/Mask\ Group.svg);
+  background-repeat: no-repeat;
+  background-color: transparent;
+  width: 80px;
+  height: 80px;
+  position: absolute;
+  right: 0;
+  top: 0;
+}
+
+.img4 {
+  background-image: url(images/illustration\ 2-about\ me\ mobile-2.svg);
+  background-repeat: no-repeat;
+  background-color: transparent;
+  width: 80px;
+  height: 80px;
+  position: absolute;
+  left: 0;
+  bottom: 0;
+}
+
+.batu {
+  text-align: center;
+  font-size: 40px;
+  color: #172b4d;
+  font-weight: 500;
+}
+
+.imgplacesholder {
+  height: 250px;
+  background-color: #ebf0ee;
+}
```

Comment:
- [ ] As stated by the previous reviewer kindly make sure the cards in your work section has the appropriate spacing and border as seen in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/206778388-394e6f9e-051c-4c67-854b-b6d14423b0a4.png)  |  ![image](https://user-images.githubusercontent.com/74814780/206778442-225a240c-aa4c-4247-a61f-ed5c155c584a.png)

- [ ] [OPTIONAL] Kindly add spacing between the cards in your work section to look as close as possible to your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/206780148-02b4b66a-deeb-4d5d-8f29-daa2be4320ec.png)  |  ![image](https://user-images.githubusercontent.com/74814780/206780181-1d7cef38-f9c9-4588-b40e-7b68a120a500.png)